docs/source/contrib.rst

```diff
@@ -0,0 +1,44 @@
+Dask Contrib
+============
+
+In addition to the primary `Dask GitHub Organization <https://github.com/dask>`_ there is also a secondary
+`Dask Contrib GitHub Organization <https://github.com/dask-contrib/>`_ which contains community supported
+projects which integrate with or enhance the Dask experience.
+
+These projects are owned and maintained by folks outside of the core Dask maintainer group and therefore
+the quality and stability may be different to that of the main Dask packages. However these projects are
+expected to conform to the `Dask Code of Conduct <https://github.com/dask/governance/blob/main/code-of-conduct.md>`_.
+
+The contrib organisztion exists to improve discoverability of user submitted projects.
+
+Contrib package criteria
+------------------------
+
+If you are working on a project which you think could be a great addition to the Dask Contrib org then please
+feel free to raise an issue on the `Dask Community repo <https://github.com/dask/community>`_ requesting a transfer.
+
+For a project to be accepted into Dask Contrib it must meet the following criteria:
+
+- Packages must complement or enhance Dask in some way, maybe it contains plugins, adds integration with some other library or just generally improves the Dask experience for users.
+- Packages must have at least one active maintainer.
+- Packages must be installable both from PyPI with ``pip`` and Conda Forge with ``conda``.
+- Packages must have documentation.
```

Comment:
This is pretty package-centric, I can imagine there being examples, benchmarks, or integration tests that live in dask-contrib. But I am fine with just crossing that bridge when/if we get to it.

Reply:
Would the benchmarks at https://blog.dask.org/2019/09/30/dask-hyperparam-opt#performance be appropriate? The data behind these benchmarks is at https://github.com/stsievert/dask-hyperband-comparison.

Reply:
I think it's alright to be package-centric. Things like examples and benchmarks are useful, but can also be a slippery slope if several different groups come forward with their own benchmarks. I'd prefer to keep a focus of tools, plugins, integrations, etc. for `dask-contrib` and direct examples to `dask-examples`. Though I'm certainly interested in other's thoughts too

Reply:
Good point @jrbourbeau. I only have a desire to use dask-contrib with the implementations, not the experiments. That's true for the Hyperband implementation above, and it's also true for my current project (I think [adadamp] should eventually go under dask-contrib, but I do not feel the same way about [adadamp-experiments]).

[adadamp]:https://github.com/stsievert/adadamp
[adadamp-experiments]:https://github.com/stsievert/adadamp-experiments

Reply:
This makes total sense. I am happy with the package-centric approach :)